src/components/Book.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Books from './Books';
+import AddBook from './AddBook';
+
+const BooksPage = () => (
+  <>
+    <Books />
```

Comment:
- [ ] Great job for initializing your component. Hhowever, there's little stuff to get correctly here. This BooksPage should render list of books using your Books component. It requires props to be passed and use that props in the Books component to render the value of the props on the browser. It should look something similar to this :point_down: 

```
const BooksPage = () => (
   <>
      <Books title="Book One" author="Author One" />
      <AddBook />
    </>  
)
```